LAYOUT_ENGINE.md

```diff
@@ -0,0 +1,187 @@
+#  The Kedro Viz Graph Layout Engine
+
+An engine for drawing [Kedro](https://github.com/quantumblacklabs/kedro) pipelines, visualised as a graph built from nodes connected by edges. 
```

Comment:
```suggestion
You can use this engine to [draw Kedro pipelines](https://github.com/quantumblacklabs/kedro), visualised as a graph built from nodes connected by edges. 
```

Reply:
Just sort-of removing passive-ness and making the link a bit more explicative. Feel free to further experiment with it!